docs/modules/language-reference/pages/index.adoc

```diff
@@ -655,7 +655,7 @@ Another difference is that in Pkl, object properties are late-bound. Read on to
 
 [[amends-declaration]]
 [NOTE]
-.Amends expressions vs. amends declarations
+.Amends declaration vs. amends expression
```

Comment:
What's the thinking behind re-ordering this?

Reply:
Order in heading and paragraph should be the same.